apps/oidc-provider-nest/src/main.ts

```diff
@@ -103,6 +112,11 @@ async function bootstrap() {
 
   const dir = join(__dirname, 'assets/views');
 
+  // This will setup the Mailer (gmail or ethereal)
+  // Mailer.build('gmail', mailerConfig);
+  Mailer.build('ethereal');
+  Mailer.sendPasswordResetConfirmationEmail('atharmon@tamu.edu');
```

Comment:
What's up with this hard-coded email?

Reply:
Was probably testing the functionality of the different mailing options